### DISCUSSION TITLE
v1/en/docs/look-and-feel/

### DISCUSSION THREAD

**Discussion by giscus** [razonyang/hugo-theme-bootstrap#983] [General]
# v1/en/docs/look-and-feel/

By default, a site using this theme has the default fonts, colors, and general look and feel.
However, the default scheme cannot satisfy everyone, but don’t worry, you can easily override the theme defaults, such as palette colors, fonts, syntax highlighting.

https://hbs.razonyang.com/v1/en/docs/look-and-feel/

**Comment by wildserver** [comment]
https://hbs.razonyang.com/v1/en/docs/look-and-feel/#fonts

As in the example does not work. The string should be like this: 
```scss {title="assets/main/scss/_custom.scss"}
 @import url ('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
```

**Reply by razonyang** [reply]
Thanks for pointing it out, will correct the docs later.
